src/components/DatePickerRange.react.js

```diff
@@ -1,3 +1,4 @@
+import 'react-dates/initialize';
```

Comment:
This is the new (or old, in any case the documented) way of using `react-dates`, as is on the [github page](https://github.com/airbnb/react-dates#initialize)